mne/annotations.py

```diff
@@ -39,15 +39,6 @@ class Annotations(object):
 
     Parameters
     ----------
-    onset : array of float, shape (n_annotations,)
-        Annotation time onsets relative to the ``orig_time``, the starting time
-        of annotation acquisition.
-    duration : array of float, shape (n_annotations,)
-        Durations of the annotations in seconds.
-    description : array of str, shape (n_annotations,) | str
-        Array of strings containing description for each annotation. If a
-        string, all the annotations are given the same description. To reject
-        epochs, use description starting with keyword 'bad'. See example above.
```

Comment:
why did you reorder parameters? it will break the tests

Reply:
How it will break the tests? It's not a inputs of a function, just doc of class parameters.

Reply:
@makkostya there are some tests to test documentation consistency. Changing the order of the class parameters might lead to tests screaming. (as is the case).

I think that keeping `orig_time` as the last parameter is not a big problem. It is true that `onset` depends on `orig_time` which is not yet explained. But its natural since its the last parameter of the call. I don't think any user would have troubles to find the right information is just 10 lines below.